contracts/src/helpers/PBHExternalNullifier.sol

```diff
@@ -55,6 +55,6 @@ library PBHExternalNullifier {
         (uint8 pbhNonce, uint8 month, uint16 year) = PBHExternalNullifier.decode(externalNullifier);
         require(year == BokkyPooBahsDateTimeLibrary.getYear(block.timestamp), InvalidExternalNullifierYear());
         require(month == BokkyPooBahsDateTimeLibrary.getMonth(block.timestamp), InvalidExternalNullifierMonth());
-        require(pbhNonce <= numPbhPerMonth, InvalidPbhNonce());
+        require(pbhNonce < numPbhPerMonth, InvalidPbhNonce());
```

Comment:
I might be out of the loop, can you clarify why this shouldnt be `<=`?

Reply:
because the (0..=29) is equal to 30 entries. the pbhNonces are 0 indexed.